pages/forgotpassword.tsx

```diff
@@ -0,0 +1,68 @@
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import { AuthContext } from "../context/AuthContext";
+import { getAuth, sendPasswordResetEmail } from "firebase/auth";
+import styles from "../styles/ForgotPassword.module.css";
+import ForgotPasswordMain from "../components/ResetPassword/ForgotPasswordLanding";
+import ForgotPasswordHelp from "../components/ResetPassword/ForgotPasswordHelp";
+
+
+const ForgotPassword: React.FC = () => {
+
+    const auth = getAuth();
+    const router = useRouter();
+
+    const [displayMain, setDisplayMain] = useState<boolean>(true);
+    const [email, setEmail] = useState<string>("");
+    const [error, setError] = useState<string>("");
+    
+
+    const sendResetEmail = (): void => {
+        sendPasswordResetEmail(auth, email)
+        .then(() => {
+            // Password reset email sent!
+            setDisplayMain(false)
+        })
+        .catch((e) => {
+            setError("No existing account with this email.")
+        });
```

Comment:
since we are using async/await throughout the entire app, can we convert this function to use the await keyword instead of .then

Reply:
Also, can we move this function to the AuthContext? That way anything that interacts with firebase resides in that context